pkg/cmd/pulumi/state.go

```diff
@@ -56,6 +56,7 @@ troubleshooting a stack or when performing specific edits that otherwise would r
 	cmd.AddCommand(newStateUnprotectCommand())
 	cmd.AddCommand(newStateRenameCommand())
 	cmd.AddCommand(newStateUpgradeCommand())
+	//	cmd.AddCommand(newStateMoveCommand())
```

Comment:
I think it would also be ok to add `Hidden: true` along with a comment to the cobra command instead, and save you the uncommenting here whenever you need to run locally.

Reply:
Oh TIL, I didn't know I could do that.  That's much more convenient for testing, done :heavy_check_mark: